lib/sedna/service/server/aggregation.py

```diff
@@ -241,3 +268,17 @@ async def client_info(self, request: Request):
         if client_id:
             return server.get_client(client_id)
         return WSClientInfoList(clients=server.client_list)
+
+import os
+os.environ['config_file'] = '/home/work/server.yml'
+from plato.servers import fedavg
+
+class PlatoServer(fedavg.Server):
+    def __init__(self, model=None):
+        super().__init__(model)
+
+from plato.servers import mistnet
+
+class MistnetServer(mistnet.Server):
```

Comment:
This seems to work directly with import plato

Reply:
Yes, but we want to keep transparency when the users are using Sedna. So we package the Plato server within the Sedna lib. Sedna's communication libs (transmitter) are under development @jaypume. After that, we will replace the communication libs with Sedna build-in libs.